src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -301,668 +253,23 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model, s
     public IActionResult ChangePasswordConfirmation()
         => View();
 
-    [HttpPost]
-    [AllowAnonymous]
-    [ValidateAntiForgeryToken]
-    public IActionResult ExternalLogin(string provider, string returnUrl = null)
```

Comment:
So the methods were all moved from the AccountController. So these methods are not part of the new feature.

Reply:
There's a lot of code in this and other classes, and templates and else, that are removed, and have been factored out to the new feature and various base/helper classes. My question is: did you change anything substantial in the new code files, or did you only basically copied over the code? Because any changes are not visible, unless I visually match every piece of code (or copy them over and see if anything changed) one by one, what I'd avoid.

Reply:
No. The code was mainly copied from one place to another "almost as-is". The main difference is the introduction of the options classes. We no longer access the settings from the SiteService. Instead we use in memory Options as these options would change depending on the enabled feature. For example, the RegaitrationOptions is configured when the registration feature is enabled. Also we do similar thing when external feature is enabled, we always ensure that site registration is enabled to allow external users to login.

Reply:
Also, we no longer need to check if the registration feature is enabled as we did previously. With the new approach this is better because we are using feature dependency instead of manually checking for enabled feature which is what we did in the past

Reply:
Yeah, I see the options classes, but just wanted to make sure if e.g. `if`s where switched around or something. Great, then!